docs/src/app/components/pages/components/buttons.jsx

```diff
@@ -58,9 +58,9 @@ export default class ButtonPage extends React.Component {
             desc: 'Override the inline hover color of the button\'s root element.',
           },
           {
-            name: 'label or children',
-            type: 'string (label) or HTML/React elements (children)',
-            header: 'required',
+            name: 'label',
+            type: 'string',
+            header: 'optional',
```

Comment:
That's right on @elacin. Until we do that, let's keep it this way to reinforce the idea that *one of* `label` or `children` is required.